dateparser_cli/utils.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import os
+from pathlib import Path
+import logging
+
+DEFAULT_UNIX_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", '~/.cache')
+DEFAULT_WIXDOWS_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", os.path.join(str(Path.home()), "AppData", "Roaming"))
+DEFAULT_DIR_NAME = os.environ.get("DEFAULT_DIR_NAME", 'date_parser_models')
+
+date_parser_model_home = None
+
+if sys.platform.startswith('win'):
+    # For Windows :
+    date_parser_model_home = os.path.expanduser(
+        os.path.join(
+            DEFAULT_WIXDOWS_CACHE_DIR, DEFAULT_DIR_NAME
+        )
+    )
+else:
+    # UNIX & OS X :
+    date_parser_model_home = os.path.expanduser(
+        os.path.join(
+            DEFAULT_UNIX_CACHE_DIR, DEFAULT_DIR_NAME
+        )
+    )
+
+
+def check_data_model_home_existance():
```

Comment:
I think we could improve here the naming, because with the word "check", it seems that only is checking. I would suggest to change it to `create_data_model_home()` as this is not only checking but creating, and IMHO the creation part is more relevant.

Reply:
I to believe the new make is better.
Thanks, I have made the changes 👍🏻